src/summary/profile/identifiers-card.component.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import SummaryCard from "../cards/summary-card.component";
+import SummaryCardRow from "../cards/summary-card-row.component";
+import SummaryCardRowContent from "../cards/summary-card-row-content.component";
+import HorizontalLabelValue from "../cards/horizontal-label-value.component";
+import { match } from "react-router";
+import VerticalLabelValue from "../cards/vertical-label-value.component";
+
+export default function IdentifiersCard(props: IdentifiersCardProps) {
+  const labelStyles = {
+    color: "var(--omrs-color-ink-medium-contrast)"
+  };
+
+  const valueStyles = {
+    color: "var(--omrs-color-ink-high-contrast)",
+    fontWeight: 500
+  };
+
+  function isPreferred(identifier) {
+    return identifier.use === "usual";
+  }
+
+  function preferredIdFirst(identifier1, identifier2) {
+    return isPreferred(identifier1) ? -1 : isPreferred(identifier2) ? 1 : 0;
```

Comment:
Nested ternaries, especially without parentheses, are the inevitable fate of the react developer growing addicted to functional programming 😄 

At the very least, add parentheses to the inner ternary. However, also, I think that it is most readable just to avoid nested ternaries altogether, even though it's a few more lines of code:

```suggestion
  if (isPreferred(identifier1) {
    return -1
  } else if (isPreferred(identifier2)) {
    return 1
  } else {
    return 0
  }
```

Reply:
😄 thanks, I'll address this.